tests/theseus_tests/core/test_gnc_cost.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import pytest
+
+import theseus as th
+
+import torch
+from tests.core.common import BATCH_SIZES_TO_TEST
```

Comment:
This should be `from tests.theseus_tests.core.common import BATCH_SIZES_TO_TEST`.

Reply:
Yeah we no longer should have a separate test for gnc.